index.js

```diff
@@ -1,15 +1,18 @@
+// Ctl re-export from ipfsd-ctl
+exports.Ctl = require('ipfsd-ctl')
+
 exports.config = require('./config')
 exports.testAPIs = require('./test-apis')
-exports.startIpfs = require('./start-ipfs')
-exports.stopIpfs = require('./stop-ipfs')
 exports.waitForPeers = require('./wait-for-peers')
 exports.connectPeers = require('./connect-peers')
 exports.getIpfsPeerId = require('./get-ipfs-peer-id')
 exports.MemStore = require('./mem-store')
 
+// Different abstract-leveldown implementations
 const implementations = require('./implementations')
-
-const properLevelModule = implementations.filter(i => i.key.indexOf('level') > -1).map(i => i.module)[0]
+const properLevelModule = implementations
+  .filter(i => i.key.indexOf('level') > -1)
+  .map(i => i.module)[0]
```

Comment:
I'd still love to refactor some of these away, I'm not even sure if OrbitDB users are using Mongo / Redis / etc, but I don't want to take it away now that it's in.

`npm install` times suffer a bit because of including these, but they _are_ needed by multiple modules: `orbit-db-storage-adapter`, `orbit-db-keystore`, `orbit-db-cache`. Maybe another shared package like `orbit-db-leveldown-implementatins`?

Reply:
One refactor at a time. Leaving for now.